src/text-editor-registry.js

```diff
@@ -214,14 +218,40 @@ class TextEditorRegistry {
     atom.grammars.autoAssignLanguageMode(editor.getBuffer())
   }
 
-  async subscribeToSettingsForEditorScope (editor) {
+  updateEditorSettingsForLanguageMode(editor, oldLanguageMode) {
+    const newLanguageMode = editor.buffer.getLanguageMode()
+
+    if (oldLanguageMode) {
+      const newSettings = this.textEditorParamsForScope(newLanguageMode.scopeDescriptor);
+      const oldSettings = this.textEditorParamsForScope(oldLanguageMode.scopeDescriptor);
+
+      let updatedSettings = {}
```

Comment:
Minor nitpick - you could use `const` here since `updatedSettings` is never reassigned.

Reply:
:+1: will fix!